### ISSUE TITLE
Safari&Firefox: Missing breadcrumbs & header when navigate to a category

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#6601]
**Environment Details**
Spartacus: epic/accessibility branch
Browser:  Safari & Firefox
Device: mac
Happens on: http://dev-com-3.accdemo.b2c.ydev.hybris.com:4200/electronics-spa/en/USD

**Steps to Reproduce**
- Go to a Category, for eg, Webcams
- Press ENTER

**Actual Result**
The result page displayed is missing its breadcrumbs and header "4 results for Webcams" in Safari & Firefox. 
<img width="1244" alt="Screen Shot 2020-02-17 at 2 42 23 PM" src="https://user-images.githubusercontent.com/45409515/74682538-5289e580-5194-11ea-9b09-6115d4377ad2.png">

It works on Chrome though.
<img width="1158" alt="Screen Shot 2020-02-17 at 2 46 12 PM" src="https://user-images.githubusercontent.com/45409515/74682544-56b60300-5194-11ea-90da-79ac992d787c.png">
